### DISCUSSION TITLE
Confluence Import Timeline

### DISCUSSION THREAD

**Discussion by sdyarnell** [outline/outline#3357] [General] [answered]
Great product! 

Our team is moving away from Atlassian, and Outline is a GREAT solution for us. However, its viability as a solution for us is dependent on a bulk Confluence import capability.

Is there a timeline you all have in mind for the Confluence import?

**Comment by tommoor** [comment, marked as answer]
> Is there a timeline you all have in mind for the Confluence import?

~Not yet I'm afraid, but we're working on it. In the meantime you _can_ import files exported from Confluence in their word-compatible format.~

Edit: This is now available in the cloud-hosted Outline under Settings -> Import -> Confluence and in the enterprise license.

**Reply by JoeRGitHub** [reply]
Hi,
I am interested in finding out if there is any progress in developing the solution?

**Reply by tommoor** [reply]
The feature is complete, it is available in the cloud-hosted Outline under **Settings -> Import -> Confluence** and in the enterprise license. There are currently no plans to include this functionality in the community edition.

**Reply by JoeRGitHub** [reply]
Thanks for the quick replay.
Is it also supported your solution on-prime?

**Reply by tommoor** [reply]
I'm not sure what you mean, but if you mean on getoutline.com then yes ☺️

**Reply by JoeRGitHub** [reply]
I meant you solution for Self-hosted on my own infrastructure

**Reply by tommoor** [reply]
Yes, but only with a paid enterprise license. Not in the open source (free) version

**Reply by Unknown-Players** [reply]
:( no comments
